.github/workflows/docs.yml

```diff
@@ -1,38 +1,29 @@
-name: Publish Developer Docs
+name: Docs CI
 
 on:
   push:
-    branches:
-      - main
-    paths:
-      - "docs/**"
+    branches: [main, "release/*"]
+  pull_request:
+    branches: ["*"]
+
+env:
+  nodeVersion: 12.18.0
 
 jobs:
-  checkout:
+  check:
+    name: Docs Check
     runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
```

Comment:
I would consider maybe making the timeout slightly smaller (like 15?) since prettier runs so fast very long runs are likely erroneous, but I'm not completely sure if it's a good idea.

Does changing `fail-fast` make sense if there is no matrix here? Or is it for future-proofing in case the action is run on different OSes or something?